toggles/webapp/toggles.js

```diff
@@ -26,6 +26,12 @@ module.exports = {
         'Show additions/amendments made in preparation for the reopening of the Wellcome Collection building',
       defaultValue: true,
     },
+    {
+      id: 'filterPrototype',
```

Comment:
For sanity - could we maybe name this `modalFiltersPrototype` and the component to match the name, maybe `ModalFilters`?